.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: Testing
+
+# Controls when the action will run.
+on:
```

Comment:
Travis CI used to also test the `master` branch once a week by cron. As far as I know that's possible here as well. I thin that should be added here as well.

Reply:
And every branch should be included, right?